root/release/edit/tracklist.tt

```diff
@@ -349,8 +349,17 @@
     <!-- /ko -->
 
     <div data-bind="visible: loaded() && !collapsed()">
+      <!-- ko if: tracks().length === 0 -->
+      <p style="margin: 1em">
+        <label>
+          <input id="tracks-unknown" type="checkbox" data-bind="checked: tracksUnknownToUser" />
+          [% l('I don’t know the tracklist for this medium.') %]
+        </label>
+      </p>
+      <!-- /ko -->
+
       <p class="error field-error" style="padding: 1em" data-bind="showErrorWhenTabIsSwitched: needsTracks">
-        [%~ l('A tracklist is required.') ~%]
+        [%~ l('Tracks are required. If you don’t know the tracks on this medium, but do know some info (such as the format or number of media), tick the “I don’t know” checkbox above and enter the available info. If you don’t know that either, remove all mediums.') ~%]
```

Comment:
I noticed it said to tick the "I don't know" checkbox above, but there's a checkbox labeled *exactly* that for the format which isn't what we mean, so might be less confusing to use the full label. That made the string sound slightly repetitive though, so I changed the first "the tracks on this medium" to "them" to directly follow from the previous sentence. Also replaced "mediums" with "media" at the end since we use media for the plural elsewhere in this PR. But feel free to only apply the parts you think make sense.

```suggestion
        [%~ l('Tracks are required. If you don’t know them, but do know some info (such as the format or number of media), tick the “I don’t know the tracklist for this medium” checkbox above and enter the available info. If you have no information about this release’s media, remove all media.') ~%]
```